tsdb/engine/tsm1/wal.go

```diff
@@ -129,6 +131,13 @@ func (l *WAL) enableTraceLogging(enabled bool) {
 	}
 }
 
+// setSyncDelay sets the duration to wait before fsync writes.  A value of 0
+// will cause every write to be fsync'd.  This must be called before the WAL
+// is opened.
+func (l *WAL) setSyncDelay(d time.Duration) {
```

Comment:
I was trying to avoid adding another parameter to `NewWAL` and passing in the whole `Config` was kind of messy too.